models/organization/org.go

```diff
@@ -397,8 +397,8 @@ func (org *Organization) GetOrgUserMaxAuthorizeLevel(uid int64) (perm.AccessMode
 }
 
 // GetUsersWhoCanCreateOrgRepo returns users which are able to create repo in organization
-func GetUsersWhoCanCreateOrgRepo(ctx context.Context, orgID int64) ([]*user_model.User, error) {
-	users := make([]*user_model.User, 0, 10)
+func GetUsersWhoCanCreateOrgRepo(ctx context.Context, orgID int64) (map[int64]*user_model.User, error) {
```

Comment:
What I've just noticed: We should rewrite the call to this method in `models/activities/notification.go#148`, as the loop can use the value directly.
I think this can also be done in this PR and doesn't need a separate PR.

Reply:
https://github.com/go-gitea/gitea/pull/22560/commits/3c7dcbea2fde9cff07c4d737824215b95b6bf9d2